cupidcoder/reactify-questioner
# 165539241 Configure testing on the application

#### Description
Adds testing with `Jest` and `enzyme` to the application.

#### Type of change
Non-breaking change which adds a development functionality.

#### How should this be manually tested?
- Install dependencies to get the latest packages using `npm install`
- run `npm run test` to see the default and only test passing


#### Checklist:
- [x] configure jest and enzyme to project

#### What are the relevant pivotal tracker stories?
[#165539241](https://www.pivotaltracker.com/story/show/165539241)
